uint8[] track_map 
uint32 car_position_x
uint32 car_position_y